src/dbc.cpp

```diff
@@ -147,20 +172,24 @@ namespace libdbc {
 				bool is_multiplexed = false; // No support yet
 				uint32_t start_bit = std::stoul(match.str(3));
 				uint32_t size = std::stoul(match.str(4));
-				bool is_bigendian = (std::stoul(match.str(5)) == 1);
+				bool is_bigendian = (std::stoul(match.str(5)) == 0);
 				bool is_signed = (match.str(6) == "-");
 				// Alternate groups because a group is for the decimal portion
-				double factor = std::stod(match.str(7));
-				double offset = std::stod(match.str(9));
-				double min = std::stod(match.str(11));
-				double max = std::stod(match.str(13));
+                double factor;
+                fast_float::from_chars(match.str(7).data(), match.str(7).data() + match.str(7).size(), factor);

```

Comment:
I think i commented on the bitstream but do we need this library as well? If this gets you going I am good with it. I might come back and rework it out if the single header fails with the new libraries.

Reply:
the problem is you have to convert the values to a double, but std::stod considers localization. I did not find another way, and this library is way faster than the std lib, and is only a single header

Reply:
Ah gotcha. I didn't realize that #8 mattered with this. Learned something new on that.